vita3k/modules/SceVshBridge/SceVshBridge.cpp

```diff
@@ -746,7 +748,8 @@ EXPORT(int, vshSblSsIsDevelopmentMode) {
 }
 
 EXPORT(int, vshSblUtMgrHasComTestFlag) {
-    return UNIMPLEMENTED();
+    STUBBED("");
+    return 1;
```

Comment:
I don't remember, is this function expected to return 1 on retail firmware?

Reply:
> I don't remember, is this function expected to return 1 on retail firmware?

Without it vsh no works, so i no know, pri cess have speaked about it too

Reply:
For example, looking at the disassembly, this function will return an error if the process executing it is not a system app.

Reply:
I see, i need check it

Reply:
> For example, looking at the disassembly, this function will return an error if the process executing it is not a system app.

ok, yes it return -0x7ffeffa8 if is not system app

and after it call kernel version of function and change value to 1 if is not on 0 🤔

Reply:
My current pr opf decrypt fself in a fly get help for correct implement this pr